docs/modules/airflow/examples/example-airflow-gitsync.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: airflow.stackable.tech/v1alpha1
+kind: AirflowCluster
+metadata:
+  name: airflow
+spec:
+  image:
+    productVersion: "2.4.1"
+    stackableVersion: "23.4.0-rc3"
+  executor: CeleryExecutor
+  loadExamples: false
+  exposeConfig: false
+  credentialsSecret: test-airflow-credentials # <1>
+  clusterConfig:
+    dagsGitSync: # <2>
+      - name: git-sync # <3>
+        repo: https://github.com/stackabletech/airflow-operator # <4>
+        branch: "main" # <5>
```

Comment:
Having an example how to check out a tag would be great. One idea would be to add needed attributes in this example and comment them out with some comment that this is an example on how to specify a tag